app/models/user.rb

```diff
@@ -1,6 +1,6 @@
 class User < ApplicationRecord
   validates :name, presence: true
-  validates :email, presence: true, uniqueness: true
+  validates :email, presence: true, uniqueness: true, format: { with: URI::MailTo::EMAIL_REGEXP }
```

Comment:
I have not seen this `format: { with: URI::MailTo::EMAIL_REGEXP }` code before and excited to see what it does!